packages/metro/src/Server.js

```diff
@@ -456,6 +456,7 @@ class Server {
       if (process.env.REACT_NATIVE_ENABLE_ASSET_CACHING === true) {
         res.setHeader('Cache-Control', 'max-age=31536000');
       }
+      res.setHeader("Content-Type", mime.lookup(path.basename(assetPath)));
```

Comment:
We already have this in `_rangeRequestMiddleware` - let's deduplicate the logic to set the header, and ensure range requests still work after this change.